charts/rasa-x/values.yaml

```diff
@@ -99,6 +99,10 @@ rasax:
     # annotations for the service
     annotations: {}
 
+  # additional labels can be used to map organizational structures onto system objects
```

Comment:
The comment should start with the name of the property that it describes, more: https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml